wordpress-mobile/WordPress-iOS
# Fix an issue where some cells in Activity Log would be visibly smaller for no good reason.

Fixes #9297

This is a tiny one — I went trough some of my half-finished PRs over the weekend and here are the fruits ;P

To test:
Truth be told, I think it's been halfway fixed by the backend almost never returning items with empty summary label, but to force it to happen:

Go to `ActivityTableViewCell.swift`, and replace line 26 with:

```swift
var rand = SystemRandomNumberGenerator()

        if rand.next() % 2 == 0 {
            summaryLabel.text = activity.summary
        } else {
            summaryLabel.text = nil
        }
```

Then, visually verify that cells with one of the labels empty still have the same height.

Update release notes:
